### PR TITLE
added reacting to poll

### PR DESCRIPTION
**Pull request by featrd3**
Added recognition of reactions to poll:
Added function for handling binary voting (uses thumbsup and thumbsdown emotes as means of voting for and against), mainly as means of acccepting input displayed in embed message. Other reactions are ignored in voting process.

### CODE REVIEW

File: BotCommands/LoLCommands.cs [hunk 1]
```diff
@@ -26,10 +31,19 @@ public async Task HelpLol(CommandContext ctx)
         [Description("Requires command in format 'Map|Nick=Champion,Nick=Champion|Nick=Champion,Nick=Champion'. First '|' separates name of map from winning team, and second '|' separetes winning team from loosing team.")]
         public async Task GetGameDataLoL(CommandContext ctx, string msg)
         {
+            var interactivity = ctx.Client.GetInteractivity();
+
             var gameData = LoLGameData.GameDataFill(msg);
             var embed = LoLCommandsSupp.LolEmbedBuilder(gameData);
 
-            await ctx.RespondAsync(embed: embed.Build()).ConfigureAwait(false);
+            var duration = TimeSpan.FromSeconds(10);
```
Comment (patrycjabru) [inline BotCommands/LoLCommands.cs]: You can put this "10 seconds" in the configuration file.

File: BotCommandsSupplement/CommonBotCommands.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        {
+            var result = await interactivity.CollectReactionsAsync(poll, duration).ConfigureAwait(false);
+
+            var thumbsDownReaction = result.Where(x => x.Emoji.Name.Equals("👎")).FirstOrDefault();
+            if (thumbsDownReaction != null)
+            {
+                var creatorGaveThumbsDown = thumbsDownReaction.Users.Where(x => x.Username.Equals(ctx.User.Username)).FirstOrDefault();
+                if (creatorGaveThumbsDown != null)
+                {
+                    return (false);
```
Comment (patrycjabru) [inline BotCommandsSupplement/CommonBotCommands.cs]: Those brackets are not needed here.

File: DiscordBot/Bot.cs [hunk 3]
```diff
@@ -27,16 +30,23 @@ public async Task RunAsync()
...
                 AutoReconnect = true,
                 MinimumLogLevel = Microsoft.Extensions.Logging.LogLevel.Debug
-                };
+               
+            };
         
             Client = new DiscordClient(config);
             Client.Ready += OnClientReady;
 
+            Client.UseInteractivity(new InteractivityConfiguration()
```
Comment (patrycjabru) [inline DiscordBot/Bot.cs:41]: This can be encapsulated in some well named method for example AddEmojiInteractivity() or something like this.

**Comment by featrd3** [comment]
Changed accordingly:
- moved poll duration to config file
- removed brackets from "CommonBotCommands"

Did not encapsulate "UseInteractivity", to keep it consistent with other similar bot settings in the same class.

Also fixed loading data from config file in "Bot" class to use already present functionality.

**Review by patrycjabru — APPROVED** [review]

**Pull request merged**